Day-11-Quiz-App/index.html

```diff
@@ -113,13 +119,93 @@ <h2>Custom Quizzes (Work in progress)</h2>
             customQuizContainer.style.display = "none";
 
         });
+
         quizElement.addEventListener("QuizBuilder:quit", () => {
+            // TODO: pass data of quiz that was just saved along in event, so that we only need to add one more button for the new quiz instead of rebuilding all buttons.
             quizChooser.style.display = "block";
             customQuizContainer.style.display = "block";
+            // TODO: implement dedicated QuizBuilder:save event, as we do not need to rebuild the buttons if no quiz was created.
+            displayCustomQuizElements();
         });
 
+        const displayCustomQuizElements = () => {
+            displayCustomQuizButtons(getCustomQuizzesFromLocalStorage()); // displays custom quiz buttons on initial page load

```

Comment:
The code comment you added here is not true for every case where `displayCustomQuizElements` is called so you should better remove it.